qemu/tests/qemu_guest_agent.py

```diff
@@ -343,6 +343,23 @@ def gagent_verify(self, params, vm):
         self.gagent.verify_responsive()
         logging.info(self.gagent.cmd("guest-info"))
 
+    @error_context.context_aware
+    def gagent_setsebool_verify(self, params, vm):
+        session = self._get_session(params, self.vm)
+        self._open_session_list.append(session)
+        error_context.context("Check selinux boolean value.", logging.info)
+        set_selinux_bool_cmd = self.params['setsebool_cmd']
+        get_selinux_bool_cmd = self.params['getsebool_cmd']
+        set_selinux_bool_guest = session.cmd(set_selinux_bool_cmd).strip()
+        value_selinux_bool_guest = session.cmd_output(get_selinux_bool_cmd).strip()
+        if value_selinux_bool_guest != 'on':
+            self.test.error("Set selinux boolean virt_qemu_ga_read_nonsecurity_files failed.\n"
```

Comment:
test.error() is okay.

Reply:
The 'self' looks like to add to.
E0602:356,12: QemuGuestAgentTest.gagent_setsebool_on: Undefined variable 'test'